packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx

```diff
@@ -0,0 +1,100 @@
+import DrawerToolbar from '@/components/Common/DrawerToolbar';
+import { MenuLinkButton } from '@/components/Common/Layout/partials/Sidebar/MenuLinkButton';
+import { sidebarLinks } from '@/constants/side-links';
+import React, { useEffect, useRef, useState } from 'react';
+import {
+  accordionItemContentStyle,
+  accordionItemTitleStyle,
+  infoAccordionWrapper,
+  linksBoxStyle,
+} from './styles.css';
+
+import { useIsMatchingMediaQuery } from '@/hooks/use-is-mobile-media-query';
+import { Accordion, AccordionItem } from '@kadena/react-ui';
+import { breakpoints } from '@kadena/react-ui/styles';
+import useTranslation from 'next-translate/useTranslation';
+import { useRouter } from 'next/router';
+import type { FC } from 'react';
+
+export interface InfoSections {
+  question: string;
+  content: string;
+}
+
+export interface IRightInfoSidebarProps {
+  infoSections: InfoSections[];
+  sidebarOpen?: boolean;
+}
+
+export const RightInfoSidebar: FC<IRightInfoSidebarProps> = ({
+  infoSections,
+  sidebarOpen,
+}) => {
+  const { t } = useTranslation('common');
+  const router = useRouter();
+  const isMediumScreen = useIsMatchingMediaQuery(`${breakpoints.lg}`);
+
+  console.log(isMediumScreen);
+
+  const [initialItem, setInitialItem] = useState<{ item: number } | undefined>(
+    undefined,
+  );
+
+  const drawerPanelRef = useRef<HTMLElement | null>(null);
+
+  useEffect(() => {
+    if (isMediumScreen) {
```

Comment:
We can not open the sidebar on mobile devices?

Reply:
We can, this is for the initial state

Reply:
I couldn't open it when clicking the question mark icon on mobile view

Reply:
this is changed, check last commit